style/americana.js

```diff
@@ -233,8 +233,14 @@ var map = (window.map = new maplibregl.Map({
   center: [-94, 40.5], // starting position [lng, lat]
   zoom: 4, // starting zoom
   attributionControl: false,
+  pitchWithRotate: false,
+  dragRotate: false,
```

Comment:
Let's add comments that note that these disable lines can be removed once #111 is resolved.

Reply:
good call.  will do